src/include/cell_inline.h

```diff
@@ -1163,7 +1163,12 @@ static inline void
 __wt_cell_unpack_addr(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL *cell,
   WT_CELL_UNPACK_ADDR *unpack_addr)
 {
-    WT_IGNORE_RET(__wt_cell_unpack_safe(session, dsk, cell, unpack_addr, NULL, NULL));
+    WT_DECL_RET;
+
+    ret = __wt_cell_unpack_safe(session, dsk, cell, unpack_addr, NULL, NULL);
```

Comment:
I like the change of validating the `unpack`. I believe that we are not validating the `pack` part of the code also. If there is a problem in `pack` we fail to `unpack` it. How about checking the return value of the `pack` part of the code also?

Reply:
Yep, we're not validating pack right now. The motivation was that we believed the issues we were seeing came from the unpacking side, but if you think it makes sense to validate pack, I can add that in.

Reply:
If it is not a big change to add validation for `pack`, I want it to be done, otherwise, it can be done seperately.

Reply:
No worries, I've added these in 5caee86.